src/main/groovy/org/shipkit/internal/gradle/exec/ShipkitExec.java

```diff
@@ -31,6 +31,7 @@ public void execute(ExecSpec spec) {
                     LOG.lifecycle("  " + execCommand.getDescription() + ":\n    " + StringUtil.join(execCommand.getCommandLine(), " "));
                 }
             });
+            LOG.lifecycle("  External process {} completed.", execCommand.getLoggingPrefix().trim());
```

Comment:
Btw, does Gradle log automatically the error message (and return code) in a case of failure?